scripts/GetClrDbg.sh

```diff
@@ -1,15 +1,42 @@
 #! /bin/bash
 
-script_dir=`dirname $0`
+script_dir="$( cd $( dirname "$0" ); pwd )"
```

Comment:
Can you provide the one-liner?

For me using pushd/popd puts the pushd stack on the variable.